.github/BOTMETA.yml

```diff
@@ -564,6 +564,7 @@ files:
     maintainers: $team_oracle
     support: community
   $modules/cloud/ovirt/: *ovirt
+    migrated_to: ovirt.ovirt_collection
```

Comment:
Hi, we need per file (not just directory here), search for other migrated_to entries in this file.

Reply:
Thank you for the information. Just send the patch if I should update anything please let me know.